doc/source/reference/random/bit_generators/mt19937.rst

```diff
@@ -1,9 +1,7 @@
-Mersenne Twister (MT19937) 
+Mersenne Twister (MT19937)
 --------------------------
 
-.. module:: numpy.random.mt19937
-
-.. currentmodule:: numpy.random.mt19937
+.. currentmodule:: numpy.random
```

Comment:
Are you sure current module needs to change?  Doc render looks right. The module isn't private (`numpy.random.mt19937`).

Reply:
Same comment for the other bit generators.

Reply:
It wasn't *my* intent to make it private.  There are use cases where someone wants to consuma a bit generator but doesn't want anything else (.e.g writing their own low-level Generator-like objects).  Should they be directed to numpy.random.__init__. Also, there are a lot of intentionally private parts to random, so I think there was some consideration about what should and shouldn't be private here (or what should be stable and what is not).